vector/build.gradle

```diff
@@ -327,6 +327,9 @@ android {
 
             applicationId "fr.gouv.rie.tchap"
             versionName "${versionMajor}.${versionMinor}.${versionPatch}${getTchapVersionSuffix()}" + "_b"
+
+            // Tchap: Show labs menu item only in btchap
+            resValue "bool", "labs_menu_item_visible", "true"
```

Comment:
You will probably say that I'm quibbling, but could you rename it to something like "show_labs_settings"? It is not really a menu item but a whole setting category. :)

Reply:
i used the same format like for developper mode :/

Reply:
labs_settings_visible ?